install/wipe_ncn_disks_for_reinstallation.md

```diff
@@ -178,14 +208,18 @@ RAIDs, zeroing the disks, and then wiping the disks and RAIDs.
 
     1. Master nodes.
 
+        Stop the etcd service on the master node before unmounting /var/lib/etcd
+
         ```bash
-        ncn-m# umount -v /var/lib/etcd /var/lib/sdu
+        ncn-m# systemctl stop etcd.service
+        ncn-m# umount -v /run/lib-etcd /var/lib/etcd /var/lib/sdu
+
         ```
 
     2. Worker nodes.
 
         ```bash
-        ncn-w# umount -v /var/lib/containerd /var/lib/kubelet /var/lib/sdu
+        ncn-w# umount -v /run/containerd /run/lib-containerd /var/lib/containerd /var/lib/kubelet /var/lib/sdu
```

Comment:
This might've worked, but it isn't ideal. This is umounting the parts of the containerd overlayFS, ahead of the overlayFS itself. Should be reversed to ensure the overlayFS umounts first and can sync any work it has pending before we yank the underlying persistent storage mount.

Suggestion:
```
ncn-w# umount -v /var/lib/kubelet /var/lib/sdu /run/containerd /var/lib/containerd /run/lib-containerd
```

Reply:
Changed order.